src/utils/utils_req.h

```diff
@@ -92,15 +92,6 @@ int ocf_req_new_extended(struct ocf_request **out_req, ocf_queue_t queue,
 int ocf_req_new_discard(struct ocf_request **out_req, ocf_queue_t queue,
 		ocf_core_t core, uint64_t addr, uint32_t bytes, int rw);
 
-/**
- * @brief Get number of allocated requests
- *
- * @param cache OCF cache instance
- *
- * @return Number of allocated requests
- */
-uint32_t ocf_req_get_allocated(struct ocf_cache *cache);
```

Comment:
I'm not sure if SPDK doesn't need it for their OCF integration. We need to confirm it before we remove this function from our API.

Reply:
SPDK team confirmed they won't be needing this. At the moment of stopping cache top bdevs are closed and have all IOs completed. Bottom bdevs are still open and can receive I/O. Outstanding backfill requests are drained in detach step where we freeze and wait for metadata ref counter.